<launch>
  <arg name="start_frame" default="0"/>
  <arg name="bag" default=""/>
  <arg name="results" default="$(env HOME)/Desktop/results.txt"/>

  <!-- <arg name="bag" default="$(env HOME)/Workspace/data/EuRoC/mh1.bag"/>
  <arg name="results" default="$(env HOME)/Workspace/spline_vio_tests/results/SplineVIO/mh1.txt"/> -->

  <rosparam command="load" file="$(find spline_vio)/calibs/EuRoC/calib.yaml" ns="spline_vio"/>

  <node name="spline_vio" pkg="spline_vio" type="spline_vio_node" output="screen" required="true">
    <param name="imu_topic" value="/imu0" />
    <param name="cam_topic" value="/cam0/image_raw" />
    <param name="calib" value="$(find spline_vio)/calibs/EuRoC/camera0.txt" />
    <param name="bag" value="$(arg bag)" />
    <param name="start_frame" value="$(arg start_frame)" />
    <param name="results" value="$(arg results)" />
    <!-- <param name="nogui" value="true" /> -->
    <param name="weight_imu_dso" value="6" />
  </node>
  </launch>